api/catalog/management/commands/validateopenapischema.py

```diff
@@ -1,42 +0,0 @@
-from argparse import ArgumentParser
-from pathlib import Path
-
-from django.core.management import BaseCommand, call_command
-
-from openapi_spec_validator import validate_spec
-from openapi_spec_validator.readers import read_from_filename
-from openapi_spec_validator.validation import openapi_v2_spec_validator
-
-
-class Command(BaseCommand):
-    def add_arguments(self, parser: ArgumentParser):
-        parser.add_argument(
-            "--output-dir",
-            default=Path(".").absolute(),
-            help=(
-                "The direction into which to output the spec file. "
-                "Defaults the the current working directory."
-            ),
-        )
-
-    def handle_validation(self, spec_dict: dict):
-        """Validate the spec and raise an error if the spec is not valid."""
-
-        validate_spec(spec_dict, validator=openapi_v2_spec_validator)
-
-        invalid_paths = [p for p in spec_dict["paths"].keys() if not p.endswith("/")]
-
-        assert invalid_paths == [], (
-            "All paths must end with a trailing slash. "
-            f"Found {len(invalid_paths)} missing a trailing "
-            f"slash: {', '.join(invalid_paths)}"
-        )
```

Comment:
@dhruvkb I've just realised that removing this management command accidentally ended us validating the trailing slashes. I didn't catch this during the PR review :cry: 

I created this issue to follow up on this: https://github.com/WordPress/openverse/issues/2452